executor-scripts/linux/static

```diff
@@ -39,10 +39,57 @@ configure_addresses() {
 	done
 }
 
+is_busybox() {                                                                   
+        local a                                                                  
+        for a in $(ls -l $(which ip)); do                                        
```

Comment:
I believe this might be simpler and safer:
```
binary=$(basename $(readlink -f $(which ip)))
if [ "${binary}" = "busybox" ]; then
    return 0
fi

return 1
```